src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+      integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Todo List</title>
+  </head>
+  <body>
+    <header>
+      <div>Today's To Do</div>
+      <div><a href="#"></a><i class="fas fa-sync"></i></div>
+    </header>
+    <form>
+      <input type="text" class="todo-input" placeholder="Add to your list..." />
+      <button class="todo-button" type="submit">
+        <i class="fas fa-level-down-alt"> </i>
+      </button>
+    </form>
+    <div class="todo-container">
+      <ul class="todo-list"></ul>
+    </div>
+    <div class="footer"><a href="#"> Clear all completed</a></div>
```

Comment:
- [ ] The app components are not aligned and don't match the given design. I would suggest as a starting point to have all of them in a parent container and then make all your components width 100% of the parent.

![Screenshot from 2021-09-30 08-38-49](https://user-images.githubusercontent.com/40503704/135409447-c1e02d09-e81a-4719-a412-79232b821b44.png)